twitter_ads/utils.py

```diff
@@ -55,3 +67,26 @@ def size(default_chunk_size, response_time_max, response_time_actual):
     scale = 1 / (response_time_actual / response_time_max)
     size = int(default_chunk_size * scale)
     return min(max(size, 1), default_chunk_size)
+
+
+def entity_ids(data):
```

Comment:
is this necessary? rather, does this need to be part of the SDK or just something we expect users to handle?